python/electrostatic_force.py

```diff
@@ -0,0 +1,362 @@
+                    text4=canvas.create_text(m,n,text=f"{charge2val}")
+                    line3=canvas.create_line(x_axis,y_axis,a1,n,fill="black",width="2")
+                    f,g=compute.mp(x_axis,y_axis,a1,n)
+                    distnew=int((compute.liner(x_axis,y_axis,a1,n))//60)
+                    distancenew=canvas.create_text(f,g,text=f"{distnew}")
+                    x11=m
+                    y11=n
+                except NameError or UnboundLocalError:    
+                    if flagsmall==0:  
+                        canvas.delete(distance)
+                        canvas.delete(line2)
+                        canvas.delete(oval2)
+                        canvas.delete(text2)
+                        oval4=canvas.create_oval(a1,b1,a2,b2,fill="orange")
+                        text4=canvas.create_text(m,n,text=f"{charge2val}")
+                        line3=canvas.create_line(x_axis,y_axis,a1,n,fill="black",width="2")
+                        x11=m
+                        y11=n  
+                        f,g=compute.mp(x_axis,y_axis,a1,n)
+                        distnew=int((compute.liner(x_axis,y_axis,a1,n))//60)
+                        distancenew=canvas.create_text(f,g,text=f"{distnew}")
+                    elif flagsmall==1:
+                        canvas.delete(line3)
+                        canvas.delete(oval2)
+                        canvas.delete(text2)
+                        canvas.delete(distancenew)
+                        oval4=canvas.create_oval(a1,b1,a2,b2,fill="orange")
+                        text4=canvas.create_text(m,n,text=f"{charge2val}")
+                        line3=canvas.create_line(x_axis,y_axis,a1,n,fill="black",width="2")
+                        x11=m
+                        y11=n
+                        f,g=compute.mp(x_axis,y_axis,a1,n)
+                        distnew=int((compute.liner(x_axis,y_axis,a1,n))//60)
+                        distancenew=canvas.create_text(f,g,text=f"{distnew}")
+        bamchik=i.get()
+        bumchik=mango.get()
+        z=compute.getter(bamchik)        
+        force=compute.forcecalc(charge1val,charge2val,z,distnew,bamchik)
+        M.set(str(force))       
+    canvas.bind("<Button-1>",dad)                        
+    root1.mainloop()
+#second window ka mainloop hai confuse mat hona  
+############################################################################################################################################################
+#FIRST WINDOW BY AWASTHIJI 
+root=Tk()
+root.title("COULOMB'S LAW")
+root.minsize(550,350)
+root.maxsize(550,350)
+label1=Label(root,text="Charge1",font="TIMESNEWROMAN 15 bold",bg="grey",fg="black",bd=2,relief="solid")
+label1.place(x=100,y=20)
+label2=Label(root,text="Charge2",font="TIMESNEWROMAN 15 bold",bg="grey",fg="black",bd=2,relief="solid")
+label2.place(x=100,y=60)
+Charge1=IntVar()
+Charge1.set(1)
+entry1=Entry(root,textvariable=Charge1,bd=3,relief="sunken")
```

Comment:
Codacy found an issue: ['Entry' may be undefined, or defined from star imports: tkinter (F405)](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279653)